account_payment_term_extension/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
You can remove the utf-8 lines from the top in Python3. Just safe the file in utf8 format.